docs/product-assembly_running_employee_rostering_in_wb/src/main/asciidoc/wb-employee-rostering-data-model-proc.adoc

```diff
@@ -0,0 +1,44 @@
+[id='wb-employee-rostering-data-model-proc']
+= Create the data model for the employee rostering project
+
+Use this section to create the data objects required to run the employee rostering sample project in {CENTRAL}.
+
+.Prerequisite 
+You have completed the project set up described in <<wb-employee-rostering-example-setup-proc>>.
+
+.Procedure 
+. With your new project, either click *Data Object* in the project perspective, or click *Create New Asset* -> *Data Object* to create a new data object.
+. Name the first data object `Timeslot`, and select `employeerostering.employeerostering` as the *Package*.
++
+Click *Ok*.
+. In the *Data Objects* perspective, click *+add field* to add attributes to the `Timeslot` data object.
+. In the *id* field, type `endTime`.
+. Click the drop-down menu next to *Type* and select `LocalDateTime`.
+. Click *Create and continue* to add another field.
+. Add another field with the *id* `startTime` and *Type* `LocalDateTime`.
+. Click *Create*.
+. Click *Save* in the upper right corner to save the `Timeslot` data object. 
+. Click the *x* in the upper right to close the *Data Objects* perspective and return to the *Assets* menu. 
+. Using the previous steps, create the following data objects and their attributes:
```

Comment:
attributes -> fields. While "attribute" makes perfect sense here (attribute is interchangeable with field), for the sake of consistency I'd change it to field.

More important suggestion: the `fieldName: type` format is a nice shortcut. I just think that because it's used for the first time in this chapter (book?) it deserves an explanation so that the user understands that each line represents a new field where the first item of the pair is the field name and the second one is the type.